share/picongpu/tests/compileParticlePusher/include/picongpu/param/species.param

```diff
@@ -17,54 +17,74 @@
+ *  That is, the name of a particle shape is defined by the visual form of its cloud shape S,
+ *  which describes a particle's charge density distribution.
+ *  However, the charge assigned to a mesh point due to a distribution of overlapping finite-sized
+ *  particles is obtained by evaluating and summing the assignment function W for all particles.
+ *  The assignment function is obtained by convolving the cloud shape with the top-hat function.
+ *  PIConGPU only implements the assignment functions.
+ *  **NOTE, the higher order shape names are redefined with release 0.6.0 in order to provide a consistent naming:
+ *      -> PQS is the name of the 3rd order shape (instead of PCS)
+ *      -> PCS is the name of the 4th order shape (instead of P4S)
+ *      -> P4S does not exist anymore
+ *  **
+ *  - particles::shapes::CIC : 1st order, assignment function is a piecewise linear spline
+ *  - particles::shapes::TSC : 2nd order, assignment function is a piecewise quadratic spline
+ *  - particles::shapes::PQS : 3rd order, assignment function is a piecewise cubic spline
+ *  - particles::shapes::PCS : 4th order, assignment function is a piecewise quartic spline
  */
 #ifndef PARAM_PARTICLESHAPE
 #define PARAM_PARTICLESHAPE TSC
 #endif
 using UsedParticleShape = particles::shapes::PARAM_PARTICLESHAPE;
 
-/* define which interpolation method is used to interpolate fields to particle*/
-using UsedField2Particle = FieldToParticleInterpolation< UsedParticleShape, AssignedTrilinearInterpolation >;
+/** define which interpolation method is used to interpolate fields to particles
+ */
+using UsedField2Particle = FieldToParticleInterpolation<
+    UsedParticleShape,
+    AssignedTrilinearInterpolation
+>;
 
 /*! select current solver method
- * - currentSolver::Esirkepov< SHAPE, STRATEGY > : particle shapes - CIC, TSC, PCS, P4S (1st to 4th order)
+ * - currentSolver::Esirkepov< SHAPE, STRATEGY > : particle shapes - CIC, TSC, PQS, PCS (1st to 4th order)
  * - currentSolver::VillaBune< SHAPE, STRATEGY > : particle shapes - CIC (1st order) only
- * - currentSolver::EmZ< SHAPE, STRATEGY >       : particle shapes - CIC, TSC, PCS, P4S (1st to 4th order)
+ * - currentSolver::EmZ< SHAPE, STRATEGY >       : particle shapes - CIC, TSC, PQS, PCS (1st to 4th order)
  *
  * For development purposes:
- * - currentSolver::currentSolver::EsirkepovNative< SHAPE, STRATEGY > : generic version of currentSolverEsirkepov
+ * - currentSolver::EsirkepovNative< SHAPE, STRATEGY > : generic version of currentSolverEsirkepov
  *   without optimization (~4x slower and needs more shared memory)
  *
  * STRATEGY (optional):
- * - currentSolver::strategy::StridedCachedSupercells
- * - currentSolver::strategy::CachedSupercells
- * - currentSolver::strategy::NonCachedSupercells
+ * - currentSolver::StridedCachedSupercells
+ * - currentSolver::CachedSupercells
+ * - currentSolver::NonCachedSupercells
```

Comment:
Not sure if this is correct. That `strategy` namespace seems still in place

Reply:
Hmm that happened accidentally when replacing the old file in the test by the new one. As this usage is explained in the standard `species.param`, I assume it is the recommended way.
And that the documentation in this test's `species.param` has never been updated but reflects an old standard.

Reply:
`currentSolver::strategy::` must be included!

Reply:
:ok: